ci_watson/artifactory_helpers.py

```diff
@@ -39,6 +39,8 @@
                      "explode": "false",
                      "excludePatterns": []}]}
 
+TIME_NOW = datetime.now()
```

Comment:
I am trying to figure out the implication of setting this so early... But maybe it does not matter if Jenkins runs each job in the matrix in a completely isolated environment; is that so, @rendinam ?

Reply:
The idea here is to time stamp the start time of the whole build.